src/components/EventChat/EventChat.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { eventChatStyles } from './EventChat.styles';
+import { Grid } from '@material-ui/core';
+import { useSelector } from 'react-redux';
+import { ReduxState } from '../../redux/combinedReducer';
+import { DB_PATHS, getDb } from '../../lib/Firebase';
+import { EventCommentType } from '../../redux/models/EventModel';
+import CommentCard from './CommentCard';
+import SubmitCommentCard from './SubmitCommentCard';
+
+export const EventChat = (): JSX.Element => {
+  const classes = eventChatStyles();
+
+  const eventId = useSelector((state: ReduxState) => {
+    return state.events.selectedEvent;
+  });
+
+  const [eventComments, setEventComments] = useState<EventCommentType[]>([]);
+
+  /**
+   * Load the event comments for the selected event
+   */
+  useEffect(() => {
+    const unsubscribe = getDb()
+      .collection(DB_PATHS.EVENT_COMMENTS)
+      .where('eid', '==', eventId)
+      .onSnapshot((doc) => {
+        const data = doc.docs.map((value) => {
+          return value.data() as EventCommentType;
+        });
+        data.sort((a, b) => {
+          return a.lastUpdated < b.lastUpdated ? 1 : a.lastUpdated > b.lastUpdated ? -1 : 0;
```

Comment:
this can be just `b.lastUpdated - a.lastUpdated`

Reply:
Done